src/Casbin.AspNetCore.Core/Casbin.AspNetCore.Core.csproj

```diff
@@ -26,6 +26,7 @@
   <ItemGroup>
     <PackageReference Include="Casbin.NET" Version="1.3.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.1.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```

Comment:
This package is for ASP.NET Core v2.x.
We want to support 3.x or 5.x. I will use the new version to replace this.